webcompat/webhooks/ml.py

```diff
@@ -26,7 +26,7 @@ def make_classification_request(issue_number):
     return response
 
 
-def get_issue_classification(issue_number, retry_count=4, retry_sleep=3):
+def get_issue_classification(issue_number, retry_count=4, retry_sleep=7):
```

Comment:
I have experimented with sending requests using Postman, it seems like sometimes 5 sec is enough, but sometimes it takes 10-12 sec. So I'm going to set 7 for now and see if we still get errors in the logs

Reply:
That's cool. 
To better understand the data though, probably we [should monitor](https://docs.python.org/3/library/time.html#time.monotonic) success and failure. 

just after the docstring we could have something along

``` python
    log = app.logger
    log.setLevel(logging.INFO)
    start = time.monotonic()
    for _ in range(retry_count):
        response = make_classification_request(issue_number)
        if response.status_code == 202:
            time.sleep(retry_sleep)
        else:
            break
    else:
        total = round(time.monotonic() - start, 2)
        msg = f"ML FAIL: issue {issue_number} in {total} seconds with {retry_count} retries"  # noqa
        raise ConnectionError(msg)
    total = round(time.monotonic() - start, 2)
    msg = f"ML OK: issue {issue_number} in {total} seconds with {retry_count} retries"  # noqa
    log.info(msg)
    return response.json()
```

With this we have an opportunity to create a dashboard or to grep the data to graph them and understand in a couple of weeks. 

What do you think? It doesn't have to be exactly this. There might be better way to organize the code.

Reply:
Thanks for providing the code! I've adjusted it a bit, to save the current retry count